From e1d7a4c2387498719ee5eb67e837fcd30b15461c Mon Sep 17 00:00:00 2001
From: Michal Simek <michal.simek@xilinx.com>
Date: Fri, 24 Jul 2020 09:48:27 +0200
Subject: [PATCH 1460/1852] PCI: xdma: Use SPDX license header

commit 7f8644455bbb7293416402e0aa81910b078f7d30 from
https://github.com/Xilinx/linux-xlnx.git

Convert text license to SPDX as was done by other drivers.

Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/pci/controller/pcie-xdma-pl.c | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/drivers/pci/controller/pcie-xdma-pl.c b/drivers/pci/controller/pcie-xdma-pl.c
index 0ac9376ccfaf..3f5b840eec3a 100644
--- a/drivers/pci/controller/pcie-xdma-pl.c
+++ b/drivers/pci/controller/pcie-xdma-pl.c
@@ -1,11 +1,8 @@
+// SPDX-License-Identifier: GPL-2.0-only
 /*
  * PCIe host controller driver for Xilinx XDMA PCIe Bridge
  *
  * Copyright (C) 2017 Xilinx, Inc. All rights reserved.
- *
- * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License 2 as published by
- * the Free Software Foundation.
  */
 
 #include <linux/interrupt.h>
-- 
2.31.1

